web/styles/settings.css

```diff
@@ -1406,6 +1406,16 @@ $option_title_width: 180px;
             }
         }
 
+        .collapse-settings-btn,
+        .collapse-settings-btn > .icon {
```

Comment:
Let's not use super generic selector names like `.icon`; we want every class to be globally unique in the app. Use `#toggle_collapse_chevron` or maybe remove that ID and have a `collapse-settings-button-icon` class. (`btn` is a bootstrap-ism that we don't want to continue).

Reply:
[Fixed](https://github.com/zulip/zulip/compare/6a953403c2f6226389f18d600468ba9174257ed1..c116a1cd6f4381a857643da019b4491d25f90589). Also did the `btn` -> `button` rename for `.collapse-settings-btn`